<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysicsMainWindow</class>
 <widget class="QMainWindow" name="PhysicsMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/main.qrc">
    <normaloff>:/new/prefix1/transmit.png</normaloff>:/new/prefix1/transmit.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Contols</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="3" column="0">
        <widget class="CopterInputsWidget" name="inputsWidget" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="comboBox_2">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="updateCameraButton">
         <property name="text">
          <string>Update cameras</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>3D View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="CloudViewDialog" name="cloud" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="AdvancedImageWidget" name="imageView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu12">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionShowLog"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Ca&amp;mera</string>
    </property>
    <addaction name="actionCameraInput"/>
    <addaction name="actionCameraStart"/>
    <addaction name="actionCameraPause"/>
    <addaction name="actionCameraStop"/>
    <addaction name="actionCameraCaptureParamteres"/>
    <addaction name="separator"/>
    <addaction name="actionCameraCalibrate"/>
   </widget>
   <widget class="QMenu" name="menuJoystick">
    <property name="title">
     <string>&amp;Joystick</string>
    </property>
    <addaction name="actionJoystickParameters"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDownloadModels"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processi&amp;ng</string>
    </property>
    <addaction name="actionCameraModel"/>
    <addaction name="separator"/>
    <addaction name="actionFlowParameters"/>
    <addaction name="actionPatternDetector"/>
    <addaction name="separator"/>
    <addaction name="actionFlightController"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionNewBackend"/>
    <addaction name="actionStatistics"/>
    <addaction name="separator"/>
    <addaction name="actionWorldRedraw"/>
    <addaction name="actionRepositionCamera"/>
   </widget>
   <widget class="QMenu" name="menuRadio">
    <property name="title">
     <string>&amp;Radio</string>
    </property>
    <addaction name="actionRadioParameters"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStartSimulation"/>
   </widget>
   <addaction name="menu12"/>
   <addaction name="menuView"/>
   <addaction name="menuCamera"/>
   <addaction name="menuJoystick"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuRadio"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_3">
       <property name="text">
        <string>connect to real device</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/main.qrc">
         <normaloff>:/new/prefix1/connect.png</normaloff>:/new/prefix1/connect.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="connetToVirtualButton">
       <property name="text">
        <string>connect to virtual</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/main.qrc">
         <normaloff>:/new/prefix1/server_connect.png</normaloff>:/new/prefix1/server_connect.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>AUTOPILOT</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/main.qrc">
         <normaloff>:/new/prefix1/robot.png</normaloff>:/new/prefix1/robot.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>imTest</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/main.qrc">
         <normaloff>:/new/colors/colors/color_blue.png</normaloff>:/new/colors/colors/color_blue.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="JoyButton">
       <property name="text">
        <string>JoyStick mode</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/main.qrc">
         <normaloff>:/new/prefix1/controller.png</normaloff>:/new/prefix1/controller.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output from JS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="iiOutputSlider">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output from II</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logDockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="LoggerWidget" name="logWidget"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/info_rhombus.png</normaloff>:/new/prefix1/info_rhombus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionCameraInput">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/webcam_add.png</normaloff>:/new/prefix1/webcam_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Camera Input</string>
   </property>
  </action>
  <action name="actionCameraStart">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/play.png</normaloff>:/new/prefix1/play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionCameraPause">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/pause.png</normaloff>:/new/prefix1/pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="actionCameraStop">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/stop_2.png</normaloff>:/new/prefix1/stop_2.png</iconset>
   </property>
   <property name="text">
    <string>St&amp;op</string>
   </property>
  </action>
  <action name="actionJoystickParameters">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/joystick_error.png</normaloff>:/new/prefix1/joystick_error.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Parameters</string>
   </property>
  </action>
  <action name="actionCameraModel">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/webcam_add.png</normaloff>:/new/prefix1/webcam_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Camera Model</string>
   </property>
  </action>
  <action name="actionNewBackend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/flag_start.png</normaloff>:/new/prefix1/flag_start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Backend</string>
   </property>
   <property name="toolTip">
    <string>New Backend</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/application_view_list.png</normaloff>:/new/prefix1/application_view_list.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Log</string>
   </property>
  </action>
  <action name="actionCameraCaptureParamteres">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/cctv_camera.png</normaloff>:/new/prefix1/cctv_camera.png</iconset>
   </property>
   <property name="text">
    <string>Ca&amp;mera Capture Paramteres</string>
   </property>
  </action>
  <action name="actionCameraCalibrate">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/2cam-blend-b.png</normaloff>:/new/prefix1/2cam-blend-b.png</iconset>
   </property>
   <property name="text">
    <string>Cam&amp;era Calibrate</string>
   </property>
  </action>
  <action name="actionRadioParameters">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/radio_modern.png</normaloff>:/new/prefix1/radio_modern.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Radio Parameters</string>
   </property>
  </action>
  <action name="actionFlightController">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/calculator.png</normaloff>:/new/prefix1/calculator.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Flight Controller</string>
   </property>
  </action>
  <action name="actionPatternDetector">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/fingerprint.png</normaloff>:/new/prefix1/fingerprint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pattern Detector</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/monitor.png</normaloff>:/new/prefix1/monitor.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Statistics</string>
   </property>
  </action>
  <action name="actionFlowParameters">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/filter.png</normaloff>:/new/prefix1/filter.png</iconset>
   </property>
   <property name="text">
    <string>F&amp;low Parameters</string>
   </property>
  </action>
  <action name="actionDownloadModels">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/down_alt.png</normaloff>:/new/prefix1/down_alt.png</iconset>
   </property>
   <property name="text">
    <string>Download Models</string>
   </property>
  </action>
  <action name="actionWorldRedraw">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/picture.png</normaloff>:/new/prefix1/picture.png</iconset>
   </property>
   <property name="text">
    <string>World Redraw</string>
   </property>
  </action>
  <action name="actionStartSimulation">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/theater.png</normaloff>:/new/prefix1/theater.png</iconset>
   </property>
   <property name="text">
    <string>Start Simulation</string>
   </property>
  </action>
  <action name="actionRepositionCamera">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/zoom_layer.png</normaloff>:/new/prefix1/zoom_layer.png</iconset>
   </property>
   <property name="text">
    <string>Reposition Camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CloudViewDialog</class>
   <extends>QWidget</extends>
   <header location="global">uis/cloudview/cloudViewDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdvancedImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">uis/advancedImageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CopterInputsWidget</class>
   <extends>QWidget</extends>
   <header>copterInputsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoggerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loggerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionShowLog</sender>
   <signal>toggled(bool)</signal>
   <receiver>logDockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>767</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraStart</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>startCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraStop</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>stopCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraPause</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>pauseCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraInput</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showCameraInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraCaptureParamteres</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showCameraParametersWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showAboutWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionJoystickParameters</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showJoystickSettingsWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRadioParameters</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showRadioControlWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlightController</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showFlightControllerParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPatternDetector</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showPatternDetectionParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatistics</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showStatistics()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlowParameters</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showProcessingParametersWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraModel</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>showCameraModelWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCameraCalibrate</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>calibrateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRepositionCamera</sender>
   <signal>triggered()</signal>
   <receiver>PhysicsMainWindow</receiver>
   <slot>repositionCloudCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCameraInput()</slot>
  <slot>startCamera()</slot>
  <slot>pauseCamera()</slot>
  <slot>stopCamera()</slot>
  <slot>showCameraParametersWidget()</slot>
  <slot>showCameraModelWidget()</slot>
  <slot>showGraphDialog()</slot>
  <slot>showAboutWidget()</slot>
  <slot>showJoystickSettingsWidget()</slot>
  <slot>showRadioControlWidget()</slot>
  <slot>showFlightControllerParameters()</slot>
  <slot>calibrateCamera()</slot>
  <slot>showPatternDetectionParameters()</slot>
  <slot>showProcessingParametersWidget()</slot>
  <slot>showStatistics()</slot>
  <slot>repositionCloudCamera()</slot>
 </slots>
</ui>
